CHANGELOG.md

```diff
@@ -314,6 +314,7 @@ pagination.
 * (types) [\#6195](https://github.com/cosmos/cosmos-sdk/pull/6195) Add codespace to broadcast(sync/async) response.
 * (baseapp) [\#6053](https://github.com/cosmos/cosmos-sdk/pull/6053) Customizable panic recovery handling added for `app.runTx()` method (as proposed in the [ADR 22](https://github.com/cosmos/cosmos-sdk/blob/master/docs/architecture/adr-022-custom-panic-handling.md)). Adds ability for developers to register custom panic handlers extending standard ones.
 * (store) [\#6481](https://github.com/cosmos/cosmos-sdk/pull/6481) Move `SimpleProofsFromMap` from Tendermint into the SDK.
+* (types) [\6654]https://github.com/cosmos/cosmos-sdk/pull/6654) Add godocs to types/context.go
```

Comment:
afaik we don't add documentation entries to the changelog 👍 
```suggestion
```

Reply:
Thanks for thinking of this though @AlexMapley 👍

Reply:
no problem! Thanks for gutting this change for me, appreciate the help